Userland/Libraries/LibJS/Runtime/Array.h

```diff
@@ -40,13 +40,13 @@ class Array : public Object {
 
     // Non-standard but equivalent to CreateArrayFromList.
     template<typename T, FallibleFunction<T const&> Callback>
-    static ThrowCompletionOr<NonnullGCPtr<Array>> try_create_from(VM& vm, Realm& realm, ReadonlySpan<T> elements, Callback map_fn)
+    [[nodiscard]] static NonnullGCPtr<Array> create_from(Realm& realm, ReadonlySpan<T> elements, Callback map_fn)
```

Comment:
Is this override actually used now? I'd suspect all callers are going to the override above, as this shouldn't compile due to removing the `TRY` around `map_fn` below, which has to be a `FallibleFunction`

Reply:
There are four invocations of it yeah. From Intl/Locale.cpp:
```cpp
return Array::create_from<StringView>(realm, list, [&vm](auto value) {
    return PrimitiveString::create(vm, String::from_utf8(value).release_value());
});
```

Reply:
Oh wait duh, I didn't realize there were two overloads 😅